src/app/routes/utils/fetchPageData/index.js

```diff
@@ -67,12 +67,19 @@ const handleError = e => {
   };
 };
 
-const fetchData = pathname => {
+const fetchData = (pathname, { maximumAttempts = 1 } = {}) => {
   const url = getUrl(pathname);
 
   logger.info(DATA_REQUEST_RECEIVED, { url });
 
-  return fetch(url).then(handleResponse(url)).catch(handleError);
+  return fetch(url)
+    .then(data => {
+      if (data.status !== 200 && maximumAttempts > 1) {
+        return fetchData(pathname, { maximumAttempts: maximumAttempts - 1 });
```

Comment:
I'd say we need a wait or back-off here. If Ares was down it would receive 3 times the traffic load compared to normal, probably making matters worse for the service.

Reply:
Initially, we should only make another request in the case of a 202. We should log an info level message for each retry, with url, status & pageType, so we can have adequate monitoring about how many retries Simorgh is making.

Reply:
👍 started conversation w/ ARES to see if they have any concerns